Aardvark/ARKEmailBugReporter.m

```diff
@@ -21,12 +21,13 @@
 #import "ARKEmailBugReporter.h"
 #import "ARKEmailBugReporter_Testing.h"
 
-#import "AardvarkDefines.h"
 #import "ARKDefaultLogFormatter.h"
 #import "ARKScreenshotLogging.h"
-#import "ARKLogStore.h"
 #import "ARKLogMessage.h"
 
+#import <CoreAardvark/AardvarkDefines.h>
```

Comment:
Good find! We should put imports from other modules above local imports though, right?